app/src/main/java/fr/free/nrw/commons/mwapi/ApacheHttpClientMediaWikiApi.java

```diff
@@ -83,8 +98,6 @@ public ApacheHttpClientMediaWikiApi(String apiURL) {
         });
     }
 
-    @Override
-    @NonNull
```

Comment:
Have we intentionally removed @NonNull ?

Reply:
No, I guess that was removed mistakenly while resolving merge conflicts. I'll revert

Reply:
no it was a mistake, I'll add it